packages/api/tests/api/auth/get.test.ts

```diff
@@ -3,7 +3,7 @@ import { get } from '../../../src/api/auth/get';
 describe('auth SLACK', () => {
   it('redirects to correct url for happy path', async () => {
     const fullLink =
-      'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=undefined%2Fapi%2Fauth%2Fcallback%2Fget&client_id=undefined';
+      'https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=undefined%2Fapi%2Fauth%2Fcallback%2Fslack%2F&client_id=undefined';
```

Comment:
Ooooof I didn't realize how janky this was; can you do the following? This _should_ work...


Also what the heck is that random undefined doing in there?? 🤨 
```suggestion
      `https://slack.com/openid/connect/authorize?scope=openid%20email%20profile&response_type=code&redirect_uri=${encodeURIComponent('undefined/api/auth/callback/slack')}&client_id=undefined`;
```

Also don't merge this suggestion; try it locally first and preferably even break it out as a separate var

Reply:
the undefined's are for values that normally come from env variables

Reply:
@SpencerKaiser Fixed!